### ISSUE TITLE
command line arguments : run specified file without dialog 

### ISSUE THREAD

**Issue by DannyCork** [dirkwhoffmann/virtualc64#256]
Firstly, great emulator. 

I've noticed that I can run virtualc64 from the command line like thus :
` ./VirtualC64 /Users/d/Downloads/vet.prg`

This then starts VirtualC64 and brings up a dialog box like this
![screen shot 2017-08-31 at 00 02 53](https://user-images.githubusercontent.com/3512672/29898810-c4d350de-8ddf-11e7-9305-bf352f9a64b4.png)

The reason for asking is that I am developing code in Sublime Text , compiling with C64Tas and then executing Virtualc64 with the newly compiled prg. As I'm writing continually testing a different version of a prg I'd like to avoid this dialog and make it flash into ram by default. 

Can this be achieved easily @dirkwhoffmann  ?

**Comment by dirkwhoffmann** [comment]
Here is my idea: 

Let's add an option like this: "Skip media dialog when opening files"
 
If this option ist set, the emulator will automatically: 
- create and insert a floppy disk, if it's a .d64, .g64, or .t64 file
- create and insert a tape, if it's a .tap file
- flush data into memory, if it's a .prg or .p00 file

**Comment by DannyCork** [comment]
sounds like a solid plan. Looking forward to it.

**Comment by dirkwhoffmann** [comment]
Fixed in 1.8

**Comment by mlund** [comment]
Hi!
I have a similar setup for cross-compiling and I'm after a smooth work-flow to test an executable on a freshly restarted C64. In v2.2 I just tried:

~~~
./VirtualC64 ~/code/MARKS.prg
...
D64Archive: Creating D64 archive from a PRG archive...
Disk: Encoded 61488 bits (7686 bytes) for tracks 1 to 17.
Disk: Encoded 57000 bits (7125 bytes) for track 18 (directory).
Disk: Encoded 57000 bits (7125 bytes) for tracks 19 to 24.
Disk: Encoded 53280 bits (6660 bytes) for tracks 25 to 30.
Disk: Encoded 49912 bits (6239 bytes) for tracks 31 and above.
~~~

which converts the prg file to an attached disk image and I would then do `load "*",8,1`.

Ideally I'd like to flush the prg file directly to memory, possibly from the command line, but a keyboard shortcut in an already open VirtualC64 session would also be a great option as there's little need to restart the full GUI in between runs.

PS: Great project!
